src/components/expanded-state/unique-token/UnderlyingAsset.js

```diff
@@ -51,6 +52,13 @@ export default function UnderlyingAsset({
     });
   }, [address, genericAssets, nativeCurrency, push, removeNextToLastRoute]);
 
+  const colorFromAsset = useColorForAsset(asset, color);
+
+  const columnWidth = useMemo(() => {
+    if (percentageAllocation < 2) return 4;
+    return percentageAllocation * 2;
+  }, [percentageAllocation]);
```

Comment:
Is it something we really need to memo? It is just a couple of simple calculations.
```suggestion
  const columnWidth = Math.max(4, percentageAllocation * 2);
```

Reply:
until the designers gave feedback, I wasn't sure how it would look
fixed now